internal/internal_update_test.go

```diff
@@ -169,7 +171,6 @@ func TestUpdateValidatorFnValidation(t *testing.T) {
 }
 
 func TestDefaultUpdateHandler(t *testing.T) {
-	t.Parallel()
```

Comment:
Why can't this be parallel any more?

Reply:
The workflow environment  this test shares isn't meant to be multithreaded